src/insns/cadd_32bit.adoc

```diff
@@ -0,0 +1,61 @@
+<<<
+
+[#CADDI,reftext="CADDI"]
+==== CADDI
+
+See <<CADD>>.
+
+[#CADD,reftext="CADD"]
+==== CADD
+
+include::new_encoding_note.adoc[]
+
+ifdef::cheri_v9_annotations[]
+NOTE: *CHERI v9 Note:* these instructions were called CINCOFFSET and
+ CINCOFFSETIMM.
+
+NOTE: *CHERI v9 Note:* the immediate format has changed
+endif::[]
+
+Synopsis::
+Capability pointer increment
+
+Mnemonic::
+`cadd cd, cs1, rs2` +
+`caddi cd, cs1, imm`
+
+NOTE: all mnemonics must be unique, but the intention is to always use `cadd/caddi`
```

Comment:
I think there should be a note here that the assembler should accept `add cd, cs1, <op>`?

Reply:
```suggestion
Suggested assembly syntax::
`add cd, cs1, r2` +
`add cd, cs1, immg`

NOTE: all mnemonics must be unique, but the intention is to always use `cadd/caddi`
```
Maybe like this?